packages/minuteman/buildinfo.json

```diff
@@ -4,15 +4,14 @@
     "minuteman": {
       "kind": "git",
       "git": "https://github.com/dcos/minuteman.git",
-      "ref": "54b665a30deb8ee7c3230140273e580d98826ad7",
+      "ref": "e6226c74a860d7cc1b7661862cb14f714426ba58",
       "ref_origin": "master"
     }
   },
   "sysctl": {
       "dcos-minuteman": {
-          "net.netfilter.nf_conntrack_tcp_be_liberal": "1",
-          "net.netfilter.ip_conntrack_tcp_be_liberal": "1",
-          "net.ipv4.netfilter.ip_conntrack_tcp_be_liberal": "1"
+          "net.ipv4.vs.snat_reroute": "1",
+          "net.ipv4.vs.conntrack": "1"
```

Comment:
As mentioned in the PR, we converted the code to us the IPVS datapath. Existing users will experience a seamless transition.